### PR TITLE
add sxknnn.md

### PR DESCRIPTION
**Pull request by sxknnn**
https://raw.githack.com/sxknnn/sxknnn.github.io/add-sxknnn-profile/index.html#!pages/mi/profiles/sxknnn.md
sxknnn.md

### CODE REVIEW

**Review by xyb994 — APPROVED** [review]

File: pages/mi/profiles/sxknnn.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+About Me
+----------
+I am an **Android Developer** with a passion of creating innovative and user friendly mobile applications.
+
+I use these languages for app development:
+<ul>
+  <li> JavaScript </li>
+  <li> Java </li>
+</ul>
```
Comment (xyb994) [inline pages/mi/profiles/sxknnn.md:15]: Please avoid HTML here as we are practicing with Markdown. I will look into fixing the profile examples.

**Review by xyb994 — CHANGES_REQUESTED** [review]

File: pages/mi/profiles/sxknnn.md (hunk 1 above)
Comment (dogi) [reply pages/mi/profiles/sxknnn.md:15]: agree

**Review by dogi — APPROVED** [review]
@sxknnn approve the rest

**Comment by xyb994** [comment]
closing this as the requested changes are not addressed over 2 month

**Pull request closed without merging**